.travis.yml

```diff
@@ -8,6 +8,10 @@ jdk:
 services:
   - memcached
   - redis
+install:
+  - make travis-install
+before_script:
+  - make travis-run
```

Comment:
Well done for getting Redis Cluster working on Travis!

Maybe we could extend this in future to set up Redis Sentinel as well. Currently the Redis Sentinel tests are skipped on Travis.

Reply:
This is a bit hacky because it's using deprecated `redis-trib.rb` instead of `redis-cli`. `travis-install` can be removed if redis on travis is updated to v5. 

Yeah, this can totally be expanded to support Redis Sentinel.